### PR TITLE
refactor(rxMetadata): moved rxMetadata from components to elements:

### PR DESCRIPTION
**Pull request by kevincareer427**
JIRA: https://jira.rax.io/browse/FRMW-283

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM
- [x] QE LGTM

### CODE REVIEW

**Comment by kevincareer427** [comment]
## Before
before

## After
after

**Comment by parlarjb** [comment]
LGTM

**Comment by CITguy** [comment]
@keerthip2187, the "BEFORE" screenshot is for `metadata` not `rxMetadata`, please update.

**Comment by glynnis** [comment]
@CITguy Related: do we already have a story to remove `metadata` (since it's showing old styles that shouldn't be utilized anymore)?

File: src/elements/Metadata/Metadata.meta.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "displayName": "Metadata",
+    "stability": "experimental",
```
Comment (100stacks) [inline src/elements/Metadata/Metadata.meta.json]: Please update stability to `ready`.

File: src/elements/Metadata/Metadata.meta.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "description": "",
```
Comment (100stacks) [inline src/elements/Metadata/Metadata.meta.json]: Please update description to include: `Metadata contains directives to provide consistent styling for the display of metadata information`, or something similar.

File: src/elements/Metadata/scripts/rxMeta.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-angular.module('encore.ui.rxMetadata')
+angular.module('encore.ui.elements')
 /**
  * @ngdoc directive
  * @name rxMetadata.directive:rxMeta
```
Comment (100stacks) [inline src/elements/Metadata/scripts/rxMeta.js]: Update to `elements:directive:rxMeta`.

File: src/elements/Metadata/scripts/rxMetadata.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-angular.module('encore.ui.rxMetadata')
+angular.module('encore.ui.elements')
 /**
  * @ngdoc directive
  * @name rxMetadata.directive:rxMetadata
```
Comment (100stacks) [inline src/elements/Metadata/scripts/rxMetadata.js]: Update to `elements:directive:rxMetadata`.

**Comment by CITguy** [comment]
@glynnis, `metadata` was removed in 2.x via #1659

File: src/elements/Metadata/Metadata.meta.json (hunk 1 above)
Comment (CITguy) [inline src/elements/Metadata/Metadata.meta.json]: `ready` is not a valid stability at present. Update to `stable` instead.

**Comment by kevincareer427** [comment]
@CITguy updated!

**Comment by kevincareer427** [comment]
@100stacks Changing to elements:directive:rxMetadata will not pass the test

**Comment by 100stacks** [comment]
@kevincareer427, I had a typo it should be `elements.directive:rxMetadata`.  😉

**Comment by kevincareer427** [comment]
@100stacks It's cool now, I didn't realize you were talking about the annotation. Thanks!

**Comment by 100stacks** [comment]
LGTM

**Comment by glynnis** [comment]
Design LGTM

**Pull request merged**